nixos/modules/services/misc/autorandr.nix

```diff
@@ -199,37 +196,37 @@ let
   };
 
   hookToFile = folder: name: hook:
-    nameValuePair "xdg/autorandr/${folder}/${name}" {
+    lib.nameValuePair "xdg/autorandr/${folder}/${name}" {
       source = "${pkgs.writeShellScriptBin "hook" hook}/bin/hook";
     };
   profileToFiles = name: profile:
     with profile;
```

Comment:
careful, you won't get notifications for breakage under this `with`

Reply:
should we remove it?

Reply:
We can wait for another pass through. This is a nice scoped set of changes. See my [previous](https://github.com/NixOS/nixpkgs/pull/338022#discussion_r1735282412) for the caution.